cmd/vic-machine/create/create.go

```diff
@@ -943,6 +943,16 @@ func (c *Create) Run(cliContext *cli.Context) (err error) {
 		log.SetLevel(log.DebugLevel)
 		trace.Logger.Level = log.DebugLevel
 	}
+
+	// urfave/cli will print out exit in error handling, so no more information in main method can be printed out.
+	defer func() {
```

Comment:
this code is practically a copy/paste. Have you considered moving it into a separate function?
I am also not sure how does it print a stack trace if there is no call to get a stack trace? Is there any call to a recover() somewhere?